docs/user/README.md

```diff
@@ -56,4 +56,9 @@ kyma
 
 ## Related Information
 
+> [!TIP]
+> Before you start using kyma CLI we strongly recommend adding completion formula to your shell (`bash`, `fish`, `powershell` or `zsh`).
+> Run the following to import completion formula for kyma CLI for zsh shell and enable auto-completion on `tab` button hit:
```

Comment:
What do you mean by the `tab` button hit?

Reply:
after you call it, your shell will start autompletend commands and command options when you hit tab key on the keybard